richtextfx/src/main/java/org/fxmisc/richtext/skin/TextExt.java

```diff
@@ -0,0 +1,95 @@
+package org.fxmisc.richtext.skin;
+
+import com.sun.javafx.css.converters.PaintConverter;
+import javafx.beans.property.ObjectProperty;
+import javafx.css.CssMetaData;
+import javafx.css.Styleable;
+import javafx.css.StyleableObjectProperty;
+import javafx.css.StyleableProperty;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.scene.text.Text;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TextExt extends Text {
+
+    private ObjectProperty<Paint[]> backgroundColor = null;
```

Comment:
There is another problem and that is that `com.sun.*` is not going to be visible from JDK 9 on. So unless you can implement it without `com.sun.*`, I would say go with `-fx-background-fill`. (Sure, for now you could probably get away with just giving `null` as the converter, since it is overridden anyway, but that would stop working when/if they fix the bug).